src/apps/content-editor/src/app/views/Dashboard/components/UserLatest/UserLatest.js

```diff
@@ -48,36 +48,42 @@ export function UserLatest(props) {
         {props.cardTitle}
       </CardHeader>
       <CardContent className={styles.CardContent}>
-        <ul>
-          {latest.map((log, i) => {
-            let url = "";
+        {latest.length > 0 ? (
```

Comment:
Instead of `latest.length > 0` we can take advantage of JavaScripts falsey statements and use just `latest.length`. Since this is an array and should have a length property and if no entries it will be 0 and that will equate to false.